### DISCUSSION TITLE
Welcome to athenz Discussions!

### DISCUSSION THREAD

**Discussion by abvaidya** [yahoo/athenz#1270] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 9avneet** [comment]
@abvaidya I am Little new in open source ,and I want to contribute in this project,so can you please guide a little bit,how can I start this project to accomplish my contribution.
